IHostedServiceAsAService/IHostedServiceAsAService.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.1.1" />
+    <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.1.1" />
```

Comment:
Just realised that I can reduce to just 

```
<ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.1.1" />
  </ItemGroup>
```
which then brings in both Microsoft.Extensions.Hosting and System.ServiceProcess.ServiceController. Is there a benefit to pulling both in directly?

Reply:
Actually seems then I can't resolve HostBuilder, so potentially scratch this query while I check the package graph again!

Reply:
Okay, yeah I see, brings in only Microsoft.Extensions.Hosting.Abstractions under Microsoft.AspNetCore.Hosting.

Reply:
So what should be added as nuget, **Microsoft.AspNetCore.Hosting.WindowsServices** or the other two packages ?